### DISCUSSION TITLE
Is there a way to select all on the subscription page to mark all of them watched

### DISCUSSION THREAD

**Discussion by avincent98144** [FreeTubeApp/FreeTube#4049] [Q&A]
Is there a way to select all on the subscription page to mark all of them watched? If so, I am not finding it, which seems like an obvious feature. In my case, there are hundreds of total vids that are 2-10 years old that I won't watch and no longer relevant.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
There isnt a way to do this at the moment.

See https://github.com/FreeTubeApp/FreeTube/issues/2517

**Comment by avincent98144** [comment]
Did not think so, so I’ve submitted a request, this is too important a feature to not have.
